pkg/instance/instance.go

```diff
@@ -191,6 +195,11 @@ func (i *Instance) AppsFS(appsType apps.AppType) afero.Fs {
 	panic(fmt.Errorf("Unknown application type %s", string(appsType)))
 }
 
+// DiskQuota returns the number of bytes allowed on the disk to the user.
+func (i *Instance) DiskQuota() (int64, error) {
+	return i.BytesDiskQuota, nil
```

Comment:
I feel it is premature now that you noted that. I removed the error return from the function's signature.